pkg/kubelet/cm/node_container_manager_linux.go

```diff
@@ -132,8 +132,8 @@ func enforceExistingCgroup(cgroupManager CgroupManager, cName CgroupName, rl v1.
 		return fmt.Errorf("%q cgroup is not config properly", cgroupConfig.Name)
 	}
 	klog.V(4).Infof("Enforcing limits on cgroup %q with %d cpu shares, %d bytes of memory, and %d processes", cName, cgroupConfig.ResourceParameters.CpuShares, cgroupConfig.ResourceParameters.Memory, cgroupConfig.ResourceParameters.PidsLimit)
-	if !cgroupManager.Exists(cgroupConfig.Name) {
-		return fmt.Errorf("%q cgroup does not exist", cgroupConfig.Name)
+	if err := cgroupManager.Exists(cgroupConfig.Name); err != nil {
+		return fmt.Errorf("%q cgroup does not exist: %v", cgroupConfig.Name, err)
```

Comment:
If you rename the method to `Validate()`, then you should change the error message here to be:

"%q cgroup is not valid: %v"

Otherwise, the above will end up printing out "cgroup does not exist: cgroup does not exist". Which is another reason to change the method name to `Validate()` :)